app-cdr/uif2iso/uif2iso-0.1.7c-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Convert CD images from uif (MagicISO) to iso"
+HOMEPAGE="http://aluigi.altervista.org/mytoolz.htm#${PN}"
```

Comment:
Do not use variables in HOMEPAGE. It makes the links unclickable in the editors. Vim highlights it red (i.e. as an error).